README.md

```diff
@@ -0,0 +1,186 @@
+# Hazelcast Discovery Plugin for Kubernetes
+
+This repository contains a plugin which provides the automatic Hazelcast member discovery in the Kubernetes environment.
+
+## Configuration
+
+This plugin supports **two different options** of how Hazelcast members discover each others:
+* Kubernetes API
+* DNS Lookup
+
+### Kubernetes API
+
+**Kubernetes API** mode means that each node makes a REST call to Kubernetes Master in order to discover IPs of PODs (with Hazelcast members).
+
+#### Grating Permissions to use Kubernetes API
+
+Using Kubernetes API requires granting certain permissions. Therefore, you may need to create `rbac.yaml` with the following content.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: default-cluster
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: view
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: default
+```
+  
+Then, apply `rbac.yaml`.
+
+```bash
+$ kubectl apply -f rbac.yaml
+```
+
+*Note*: You can be even more strict with the permissions and create your own Role. For details, please check the implementation of [Hazelcast Helm Chart](https://github.com/helm/charts/tree/master/stable/hazelcast) ([Role](https://github.com/helm/charts/blob/master/stable/hazelcast/templates/role.yaml), [Service Account](https://github.com/helm/charts/blob/master/stable/hazelcast/templates/serviceaccount.yaml), and [RoleBinding](https://github.com/helm/charts/blob/master/stable/hazelcast/templates/rolebinding.yaml)).
+
```

Comment:
Having such links are making maintenance hard over time. File names can be changed or they can completely be removed... I suggest to either point to a tagged versions of the file or copy/paste the content of them...